index.html

```diff
@@ -44,7 +44,163 @@ <h1>I'm Javi,<br>
+          </div>
+          <h2 class="p10px">Tonic</h2>
+          <div class="flex aic">
+            <h4 class="p10px">Canopy</h4>
+            <img src="assets/images/Counter.svg" alt="Spacer">
+            <h5 class="p10px">Back end dev</h5>
+            <img src="assets/images/Counter.svg" alt="Spacer">
+            <h5 class="p10px">2022</h5>
+          </div>
+          <p class="p10px m0">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="xul">
+            <li class="mh10px p10px border-10px bg-tag"><h6>html</h6></li>
+            <li class="mh10px p10px border-10px bg-tag"><h6>css</h6></li>
+            <li class="mh10px p10px border-10px bg-tag"><h6>javaScript</h6></li>
+          </ul>
+          <div>
+            <button>See project</button>
+          </div>
+        </div>
+      </section>
+      <section class="border-card m20px">
+        <div class="grid-container-card m20px">
+          <div>
+            <img class="border-10px fill-100" src="assets/images/Snapshoot Portfolio 4.svg" alt="Canopy project snapshot">
+          </div>
+          <h2 class="p10px">Multi-post<br>
+            stories</h2>
+          <div class="flex aic">
+            <h4 class="p10px">Canopy</h4>
+            <img src="assets/images/Counter.svg" alt="Spacer">
+            <h5 class="p10px">Back end dev</h5>
+            <img src="assets/images/Counter.svg" alt="Spacer">
+            <h5 class="p10px">2022</h5>
+          </div>
+          <p class="p10px m0">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="xul">
+            <li class="mh10px p10px border-10px bg-tag"><h6>html</h6></li>
+            <li class="mh10px p10px border-10px bg-tag"><h6>css</h6></li>
+            <li class="mh10px p10px border-10px bg-tag"><h6>javaScript</h6></li>
+          </ul>
+          <div>
+            <button>See project</button>
+          </div>
+        </div>
+      </section>
+    </article>
+    <article class="bg-box border-tr pv5">
+      <section class="p5vh">
+        <h2>About<br>
+          Myself</h2>
+        <p class="mv12px">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a class="mv12px" href="https://ly.enddesign.co/intro"><h3>Let's connect</h3></a>
+        <ul class="xul">
+          <li class="pr20px">
+            <a href="http://">
+              <img class="icon" src="assets/images/Linkedin icon.svg" alt="Linkedin profile @javierjhm">
+            </a>
+          </li>
+          <li>
+            <a href="http://">
+              <img class="icon" src="assets/images/Github icon.svg" alt="Github profile @wickathou">
+            </a>
```

Comment:
- [ ] I would suggest you add the rest of the social media icon to your project even if it won't be leading to any of your social media pages, with this you will follow the Figma design template. Currently, I can only see two of your social media icons
- kindly check this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=63ggaVULz5qmZR8p-0) to add other icons

Reply:
I am pretty disappointed that it's called "portfolio" but it is not supposed to be about us in any way.

All icons have been added now.